frontend/src/app/applications/fields/org-type.component.html

```diff
@@ -0,0 +1,48 @@
+<ng-container [formGroup]="applicantInfo" *ngIf="type === 'simple'">
+  <label for="applicant-type">Are you applying as an individual or organization?</label>
+  <ul class="usa-unstyled-list">
+    <li>
+      <input id="individual" type="radio" name="orgType" value="Person" formControlName="orgType">
+      <label for="individual">Individual</label>
+    </li>
+    <li>
+      <input id="organization" type="radio" name="orgType" value="Corporation" formControlName="orgType">
+      <label for="organization">Organization</label>
+    </li>
+  </ul>
+</ng-container>
+
+<ng-container [formGroup]="applicantInfo" *ngIf="type === 'complex'">
+  <label for="applicant-type">As an applicant, are you:</label>
+  <ul class="usa-unstyled-list">
+    <li>
+      <input id="individual" type="radio" name="orgType" value="individual" formControlName="orgType">
+      <label for="individual">Individual</label>
+    </li>
+    <li>
+      <input id="organization" type="radio" name="orgType" value="corporation" formControlName="orgType">
+      <label for="organization">Corporation</label>
+    </li>
+    <li>
+      <input id="llc" type="radio" name="orgType" value="llc" formControlName="orgType">
+      <label for="llc">Limited Liability Company</label>
+    </li>
+    <li>
+      <input id="llc" type="radio" name="orgType" value="partnership" formControlName="orgType">
```

Comment:
LLC is an option on the paper form, don't we need to include it in the digital form?

`As an applicant, are you:
__ Individual			If yes, are you a citizen of the United States? 
__ Corporation			If yes, provide a copy of your state certificate of good standing.
__ Limited Liability Company	If yes, provide a copy of your state certificate of good standing.
__ Partnership or Association	If yes, provide a copy of your partnership or association agreement.
__ State Government or Agency	(Includes state universities)
__ Local Government or Agency	(Includes high schools)
__ Nonprofit 			(Please attach a copy of your IRS Form 990)`